### PR TITLE
Add description about the common types of breadcrumbs

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds description about the common types of breadcrumbs

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/navigation-breadcrumb-trail.html

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- https://community.magento.com/t5/Magento-2-x-Programming/Breadcrumbs-in-Magento-2/td-p/101782

whatsnew
Added _Common types of breadcrumbs_ section to the [Breadcrumb Trail](https://docs.magento.com/user-guide/catalog/navigation-breadcrumb-trail.html) topic.

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Review by dobooth — CHANGES_REQUESTED** [review]
Please rewrite per our conversation.

File: src/catalog/navigation-breadcrumb-trail.md [hunk 1]
```diff
@@ -10,7 +10,33 @@ The breadcrumb trail can be configured to appear on content pages and on catalog
...
 _Breadcrumb Trail_
 
-## Remove the breadcrumbs from CMS pages
+## General types of bread crumbs
+
+Breadcrumbs can be divided into three main types, which differ in their purpose. The essence and main principles of the implementation of each type of bread crumbs are described below.
+
+### Hierarchy-based breadcrumbs
+
+This kind of breadcrumbs are based on the category hierarchy set up on the site. The chains presented tell the user at what structural level they are. In this case, each text link is intended for a page that is one level higher than the previous one.
```
Comment (leslietilling) [inline src/catalog/navigation-breadcrumb-trail.md:19]:
```suggestion
This type of breadcrumb is based on the category hierarchy set up on the site. The chains presented tell the user where in the structure they are. In this case, each text link is intended for a page that is one level higher than the previous one.
```

File: src/catalog/navigation-breadcrumb-trail.md [hunk 2]
```diff
@@ -10,7 +10,33 @@ The breadcrumb trail can be configured to appear on content pages and on catalog
...
+
+Example: `Men > Tops > Hoodies & Sweatshirts`
+
+This type of breadcrumb helps users easily see which category level they are in and provide easy navigation between catalog pages.
```
Comment (leslietilling) [inline src/catalog/navigation-breadcrumb-trail.md:23]:
```suggestion
The advantage of this type is that users can easily see which category level they are in and have easy access to navigation between catalog pages.
```

File: src/catalog/navigation-breadcrumb-trail.md [hunk 3]
```diff
@@ -10,7 +10,33 @@ The breadcrumb trail can be configured to appear on content pages and on catalog
...
+
+### History-based breadcrumbs
+
+History (or path) -based navigation is similar to the back button in a browser. This allows users to quickly return to the previous pages they have visited without changes.
+
+This type of breadcrumbs is useful when customers want to return to a previous page after selecting multiple filters on a category page.
```
Comment (leslietilling) [inline src/catalog/navigation-breadcrumb-trail.md:29]:
```suggestion
The advantage of this type is that it is most helpful when customers want to return to a previous page after selecting multiple filters on a category page.
```

File: src/catalog/navigation-breadcrumb-trail.md [hunk 4]
```diff
@@ -10,7 +10,33 @@ The breadcrumb trail can be configured to appear on content pages and on catalog
...
+
+Example: `Home > What's New > Gear > Bags`
+
+### Attribute-based breadcrumbs
+
+This type of angry crumbs displays the attributes selected on the category page. The main difference from the previous types is that the attribute-based breadcrumbs represent the filters and options you have selected in the navigation layer for certain products (price, quality, color, etc.).
```
Comment (leslietilling) [inline src/catalog/navigation-breadcrumb-trail.md:35]:
```suggestion
This type of breadcrumb displays the attributes selected on the category page. The main difference from the other types is that the attribute-based breadcrumbs represent the filters and options you have selected in the navigation layer for certain products (such as price, quality, and color).
```

**Review by leslietilling — APPROVED** [review]
I will put in style/grammar edits.

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**